### PR TITLE
feat: lottie validator

### PR DESCRIPTION
**Pull request by Aidosmf**
- moved lottie-validator logic from lottie-spec https://github.com/lottie/lottie-spec/pull/101
- adjusted & improved UI for responsiveness and similar styles
- issue: https://github.com/lottie/lottie-spec/issues/93

### CODE REVIEW

File: src/utils/validator.js
Comment (Aidosmf) [inline src/utils/validator.js:1]: planning to create an NPM package in Q4

File: src/assets/lottie.schema.json
Comment (Aidosmf) [inline src/assets/lottie.schema.json:1]: the schema will be moved to the LottieValidator NPM package

File: src/utils/validator.js
Comment (Aidosmf) [reply src/utils/validator.js:1]: it's Glax's code (only JS warnings were fixed)

File: src/assets/lottie.schema.json
Comment (mbasaglia) [reply src/assets/lottie.schema.json:1]: We'll probably want to use versioned names for the schema file once we have a separate package

**Review by mbasaglia — APPROVED** [review]

**Comment by Aidosmf** [comment]
@fmalita @fmalita I will merge it for now, so users can start using it, please raise issues if you find any

**Comment by fmalita** [comment]
Belated LGTM!

**Pull request merged**